packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -190,6 +191,13 @@ export interface FunctionProps {
    * @see https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html
    */
   reservedConcurrentExecutions?: number;
+
+  /**
+   * Event sources for this function.
+   *
+   * You can also add event sources using `addEventSource`.
+   */
+  events?: IEventSource[];
```

Comment:
I kinda feel compelled to call this `eventSources`... But I don't feel so strongly about it either.

Reply:
I wanted to align with SAM, which calls this "events"